### PR TITLE
Add Items Justification to Social Links

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Adds justify toolbar controls to Social Links, allows for justifying items in the menu (left, center, right, and space-between). This matches the navigation and buttons features.

@jasmussen - the .wp-social-links class was hard-coded with a `justify-content: flex-start;` that I removed to get this to work. I'm not sure if we need it since `flex-start` is the default.

## How has this been tested?

1. Add a social links block with some social icons
2. Confirm everything works as expected
3. Apply PR and confirm existing blocks loads and no visual change
4. Use new Justify items menu item and confirm left, center, right, space-between works in editor and while viewing.

## Screenshots

social-justify

## Types of changes

- Adds justify toolbar to social links

### CODE REVIEW

File: packages/block-library/src/social-links/save.js [hunk 1]
```diff
@@ -10,12 +10,18 @@ import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
...
+			iconColorValue,
+			itemsJustification,
+			size,
+		},
 	} = props;
 
 	const className = classNames( size, {
 		'has-icon-color': iconColorValue,
 		'has-icon-background-color': iconBackgroundColorValue,
+		[ `items-justified-${ itemsJustification }` ]: itemsJustification,
```
Comment (jasmussen) [inline packages/block-library/src/social-links/save.js:24]: Did we discuss the naming of this CSS class? I seem to vaguely recall that we did, but can't recall where we ended. I would've probably preferred something like `has-justification-left` — but maybe there was already an existing CSS class out in the wild?

**Review by jasmussen — CHANGES_REQUESTED** [review]
All looked cool right up until the end (and removing the flex-start seems fine). Before:

Screenshot 2021-02-15 at 09 01 29

After:

Screenshot 2021-02-15 at 09 02 50

Screenshot 2021-02-15 at 09 02 54
Screenshot 2021-02-15 at 09 03 00

Screenshot 2021-02-15 at 09 03 04

However it doesn't work on the frontend: this should be "space between":

Screenshot 2021-02-15 at 09 05 54

It also looks like the flex-start is still output by default on the frontend:

Screenshot 2021-02-15 at 09 03 34

It looks like the items justification CSS is not output on the frontend at all. 

I have no idea why, because the code looks right.

**Comment by mkaz** [comment]
@jasmussen I added the justification CSS to the common.scss for block-library and it all works as expected.
One thing I did notice is for centering, if the block is align=center then the social links add a justify-items center also. I'm not sure what should be done here if anything, the behavior is a bid odd when using both align and justify.

**Review by jasmussen — APPROVED** [review]
Ship it!

Screenshot 2021-02-19 at 11 01 51

Screenshot 2021-02-19 at 11 01 40

**Pull request merged**